### DISCUSSION TITLE
Backup of recipe database

### DISCUSSION THREAD

**Discussion by BencePaul** [nextcloud/cookbook#1144] [FAQ and user questions] [answered]
Hi,

Sorry if this is an obvious question, but is there a way to backup the recipe database? I have started to put everything into the app, but I have it running on an old laptop and I'm worried that when it dies, my database will die with it.

**Comment by seyfeb** [comment, marked as answer]
Currently the recipe information can be restored from the files/folders in the main recipe folder without a backup of the database by rescanning the folder from the App UI. The one big **exception are links between recipes**. Those reference the id of the other recipe which may change if you set up a new Nextcloud instance from scratch. To prevent this loss as well, you can backup the Nextcloud database (which you are probably doing already, anyway, if you backup your Nextcloud installation).

**Reply by BencePaul** [reply]
OK, great. 

Thanks @seyfeb!
